src/components/radios/index.njk

```diff
@@ -69,6 +69,12 @@ The submit button should use the `hidden` attribute and also the `u-d-no` utilit
     patternlibExample({"path": "components/radios/examples/radios-clear/index.njk"})
 }}
 
+### Radios without a border
+This variant removes the border surrounding the radio input and [label](/components/label).
```

Comment:
We should add usage guidance (@LauraC119  will provide). And the example should reflect this so it's used for the correct purpose.  We wouldn't want this used for single page question answers, so I would change the example by removing the use of `onsQuestion` and match the options up to the 'get the data' design: 

<img width="519" alt="Screenshot 2021-08-24 at 14 59 27" src="https://user-images.githubusercontent.com/43346934/130630341-8f267402-d449-4586-9ce4-a9f831b15834.png">

Reply:
I agree that the example should be aligned closer to its intended purpose; the only thing I would say against using the 'get the data' design is that it is a pattern and includes additional components like the primary call to action. Happy to include the download formats and maybe using the 'select download format' as the legend.

Reply:
Yeah I think that will do